storybook/.storybook/preview-head.html

```diff
@@ -0,0 +1,4 @@
+<link
+  rel="stylesheet"
+  href="https://fonts.googleapis.com/icon?family=Material+Icons"
```

Comment:
There are MUI icons being used elsewhere in the Storybook why does this need to be added?

Reply:
That is because to enable any icons to be added I am using the `Icon` from `@material-ui/core/Icon` to enable any icon to be added from webfonts, if there is a better way of doing this happy to change?

Reply:
Thanks, no worries 👍